go/dublintraceroute/probes/probev6/udpv6.go

```diff
@@ -86,7 +107,12 @@ func (d UDPv6) ForgePackets() []gopacket.Packet {
 			// length is 13 for the first flow, 14 for the second, etc.
 			// 13 is given by 8 (udp header length) + 5 (magic string
 			// length, "NSMNC")
-			length := 5 + dstPort - d.DstPort
+            length := uint16(len("NSMNC"))
+            if d.UseSrcPort {
+                length += srcPort - d.SrcPort
+            } else {
+                length += dstPort - d.DstPort
+            }
 			// 5 is the length of the magic string "NSMNC"
 			repeat := int(length / 5)
```

Comment:
well you have calculated the length at 110, you can move it to a variable and reuse instead of the hardcoded 5 (nitpicking, sorry :P)

Reply:
actually that `length` is modified - I can stick on hardcoded numbers though

Reply:
I mean - I can go back to using `5` instead of `len(..)` at line 110, and continue using the 5 at line 117

Reply:
Just updated, can you have a look again and tell me if this is better?

Reply:
as just discussed offline - I misunderstood you, but rolling back the inconsistency made it consistent again :P going to merge after tests pass